src/Wireable.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Livewire;
+
+interface Wireable
+{
+    public function toLivewire();
```

Comment:
in all your examples it returns an array, is there anything against a return type?

Reply:
Originally there was an array return type, but I did not want to close off the possibility of serializing to a string as well, especially if the DTO only holds one value 👍 

Once LW drops PHP <8.x support, maybe `array|string` would be good.